MotionMark/resources/debug-runner/graph.js

```diff
@@ -48,18 +48,18 @@ Utilities.extendObject(window.benchmarkController, {
 
         this._targetFrameRate = options["frame-rate"];
 
-        this.createTimeGraph(testResult, samplesWithProperties[Strings.json.controller], testData[Strings.json.marks], testData[Strings.json.controller], options, margins, size);
+        this._createTimeGraph(testResult, samplesWithProperties[Strings.json.controller], testData[Strings.json.marks], testData[Strings.json.controller], options, margins, size);
```

Comment:
I am still not sure about the use of `_` as a naming convention for private members/methods. It used to make sense when JS did not have a notion of public and private. But since JS classes now support private methods using `#`, I do not know why we have to stick with the `_`.

Reply:
We can do a wider adoption of private methods later.